src/remote-checkout/remote-checkout-action-creator.ts

```diff
@@ -91,6 +93,23 @@ export default class RemoteCheckoutActionCreator {
         });
     }
 
+    forgetCheckout(methodId: string, options?: RequestOptions): Observable<Action> {
+        return Observable.create((observer: Observer<Action>) => {
```

Comment:
Sorry again but think we need to do a `concat` of calls. You can see example here https://github.com/bigcommerce/checkout-sdk-js/blob/master/src/checkout/checkout-action-creator.ts#L37.